metadata/src/main/java/org/apache/kafka/controller/BrokersToElrs.java

```diff
@@ -159,4 +160,14 @@ BrokersToIsrs.PartitionsOnReplicaIterator partitionsWithBrokerInElr(int brokerId
         }
         return new BrokersToIsrs.PartitionsOnReplicaIterator(topicMap, false);
     }
+
+    BrokersToIsrs.PartitionsOnReplicaIterator partitionsWithElr() {
+        Map<Uuid, int[]> topicMap = new HashMap<>();
+        for (Map<Uuid, int[]> map : elrMembers.values()) {
+            if (map != null) {
```

Comment:
when would this be null? is there a particular reason we chose to use a null array instead of an empty array?

Reply:
Done, it can't be null